src/main/treeDataProviders/dependenciesTree/goDependenciesTreeNode.ts

```diff
@@ -0,0 +1,20 @@
+import { DependenciesTreeNode } from './dependenciesTreeNode';
+import { IComponentMetadata } from '../../goCenterClient/model/ComponentMetadata';
+export class GoDependenciesTreeNode extends DependenciesTreeNode {
+    private _goCenter!: IComponentMetadata;
+
+    public set goCenter(value: IComponentMetadata) {
+        this._goCenter = value;
+    }
```

Comment:
I suggest creating a constructor instead
```suggestion
```

Reply:
When we create a new  `goDependenciesTreeNode.ts` we don't have the metadata yet.
After we finish building the entire tree, we get the metadata from gocenter.
As a result, we cannot add `IComponentMetadata` to the ctor

Reply:
Ok